### ISSUE TITLE
Log Validation Loss

### ISSUE THREAD

**Issue by lapp0** [Wayfarer-Labs/owl-wms#37]
Currently some dataloaders sample a large number of data points from a small number of tarballs repeatedly, with potentially overlapping windows. We should log validation loss to wandb to ensure we're not overfitting to these samples.

Create a small validation dataset for each train dataset, and log validation metrics. Alternatively, we can just specify a tarball for validation which is held-out from training (doesn't require creating new bucket, but hacky)
